### PR TITLE
search rework

### PR DESCRIPTION
**Pull request by PavelKhomyakov**
Reworked search:
- Now search looks through discogs API.
- Shows first pic and label on page (temporary solution)

### CODE REVIEW

File: client/src/components/Search.tsx
Comment (ryanesmores07) [inline client/src/components/Search.tsx:1]: I noticed you change the variable from Search to SearchBar, if possible lets change the filename to SearchBar also to make it easier to understand. Thanks!

**Review by ryanesmores07 — CHANGES_REQUESTED** [review]

File: client/src/components/Search.tsx
Comment (PavelKhomyakov) [reply client/src/components/Search.tsx:1]: Done

File: client/src/components/index.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export { default as ModeToggle } from "./ModeToggle";
 export { default as Hero } from "./Hero";
 export { default as SearchSection } from "./SearchSection";
 export { default as FormInput } from "./FormInput";
-export { default as Search } from "./Search";
+export { default as Search } from "./SearchBar";
```
Comment (ryanesmores07) [inline client/src/components/index.ts]: also the default as SearchBar

**Review by ryanesmores07 — CHANGES_REQUESTED** [review]

File: client/src/components/index.ts (hunk 1 above)
Comment (PavelKhomyakov) [reply client/src/components/index.ts]: Done

Comment (ryanesmores07) [reply client/src/components/index.ts]: Thanks!

**Pull request merged**